ohtu2021-kvantti/WebMark
# Run-button branch

## Summary

Running of benchmarks by molecule is now done with pressing a "Run" button #104 

## How to test

Ensure that running benchmarks works and check that the changes made to "algorithm.html" are valid.
